node/runtime/src/lib.rs

```diff
@@ -253,20 +253,23 @@ parameter_types! {
 	pub const SessionsPerEra: sr_staking_primitives::SessionIndex = 6;
 	pub const BondingDuration: staking::EraIndex = 24 * 28;
 	pub const RewardCurve: &'static PiecewiseLinear<'static> = &REWARD_CURVE;
+	pub const MaxReward: Perbill = Perbill::from_percent(10);
+	// ^^^ 10% - must be equal to max_inflation, above.
```

Comment:
This is annoyingly the same value in two different places; the `curve!` macro requires a literal and we can't extract the parameters once it's created.

Reply:
The only way I see to remove this duplication is:
* make max_reward perbill an output of curve macro
* (or wait for more const in rust: all arithmetic operation needed by curve could potentially be const)
* (or make the curve build at runtime)